src/brsTypes/components/MiniKeyboard.ts

```diff
@@ -0,0 +1,23 @@
+import { FieldModel } from "./RoSGNode";
+import { Group } from "./Group";
+import { AAMember } from "./RoAssociativeArray";
+
+export class MiniKeyboard extends Group {
+    readonly defaultFields: FieldModel[] = [
+        { name: "text", type: "string", value: "" },
+        { name: "keyColor", type: "string", value: "0x000000FF" },
+        { name: "focusedKeyColor", type: "string", value: "0x000000FF" },
+        { name: "keyboardBitmapUri", type: "string", value: "" },
+        { name: "focusBitmapUri", type: "string", value: "" },
+        { name: "textEditBox", type: "node" },
```

Comment:
Currently, when we have a `node` type field in a component, we initialize it to `invalid`. In this case, should we add some logic after `registerDefaultFields` to initialize `textEditBox` to an actual valid `TextEditBox` node, since that's the behavior in RBI (I assume)?